spec/rototiller/task/rototiller_task_spec.rb

```diff
@@ -245,6 +245,23 @@ def described_verbose(verbose)
             .to_stdout
         end
       end
+
+      # confined to 'new' init method, dirty test env (rspec--)
+      context 'name default relationship', :if => (init_method == :new) do
+        it 'uses the name when there is no default' do
+          validation = 'I_AM_THE_NAME'
+          command = {:name => "echo #{validation}", :add_env => {:name => 'FOOBAR'}}
+          task.add_command(command)
+          expect{ described_run_task }.to output(/#{validation}/).to_stdout
+        end
+
+        it 'prefers the default over the name' do
+          validation = 'I_AM_THE_DEFAULT'
+          command = {:name => "echo I_AM_THE_NAME", :add_env => {:name => 'FOOBAR', :default => "echo #{validation}"}}
+          task.add_command(command)
+          expect{ described_run_task }.to output(/#{validation}/).to_stdout
+        end
```

Comment:
i don't think this belongs in rototiller_task_spec.  should be in env specs or possibly command, if env specs don't have a viable parent.  although i guess if there is no parent in env specs, here is as good a place as command.  🍧